1_fetch/cfg/dates.yml

```diff
@@ -0,0 +1,2 @@
+start: 2017-05-01
+end: 2017-05-10
```

Comment:
departing from our practice in other projects, I'd like to try getting all the config info that's unique to a storm event into a single file (viz_config.yml in the top-level folder). that would include these dates. goal is to make it really fast to spin up a viz for a new storm, which i think would be facilitated by having all the info you need to edit in just one file.

Reply:
going with jordan's suggestion in slack, our Harvey viz start/end dates were "2017-08-25" to "2017-09-02"

Reply:
I'm moving to viz_config on next push with some other changes